### DISCUSSION TITLE
（灌水）有人知道简升到五十级的材料吗

### DISCUSSION THREAD

**Discussion by 3165341164** [share121/inter-knot#3419] [其他]
这些是我攒给简的彩礼，不知道够不够，其实现在已经到190抽了
![Screenshot_20240821_150028](https://github.com/user-attachments/assets/8aa57867-9825-4b07-a1cf-5bcd2e9c2549)

**Comment by Nie7bai** [comment]
![简材料](https://github.com/user-attachments/assets/8b8354df-f5de-40cd-9f01-cccd1ee139d0)
<img src="https://img-static.mihoyo.com/communityweb/upload/0b5acb4c4e49bf8733806b11450f6c50.png" alt="三月七_Biu" width="75" >

**Reply by 3165341164** [reply]
谢谢啦好心的绳匠
